partner_credit_limit/models/sale.py

```diff
@@ -12,7 +12,7 @@ class SaleOrder(models.Model):
 
     @api.one
     def check_limit(self):
-        partner = self.partner_id
+        partner = self.partner_invoice_id
```

Comment:
Another question?

Why is the "Credit / Debit" calculation needed? 
Is "self.amount_total" not sufficient. 
Are they not always going cancel each other out?